test/groovy/PiperStageWrapperTest.groovy

```diff
@@ -163,7 +163,7 @@ class PiperStageWrapperTest extends BasePiperTest {
     @Test
     void testGlobalOverwritingExtension() {
         helper.registerAllowedMethod('fileExists', [String.class], {s ->
-            return (s == 'test_global_overwriting.groovy')
+            return (s == '.pipeline/tmp/global_extensions/test_global_overwriting.groovy')
```

Comment:
I don't fully understand those changes. Yes I see the global extensions path was prepended, does that change anything user-visible?